<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-VHEBP6NB" failures="1" tests="3" name="Error Validations" time="20.549" errors="0" timestamp="2025-05-01T23:28:42 IST">
  <testcase classname="Learnings.Tests.ErrorValidations" name="ErrorValidationsTest" time="2.629"/>
  <testcase classname="Learnings.Tests.ErrorValidations" name="productValidationCheck" time="4.304">
    <skipped/>
  </testcase> <!-- productValidationCheck -->
  <testcase classname="Learnings.Tests.ErrorValidations" name="productValidationCheck" time="2.53">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Learnings.Tests.ErrorValidations.productValidationCheck(ErrorValidations.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- productValidationCheck -->
</testsuite> <!-- Error Validations -->
